gabbi/runner.py

```diff
@@ -89,22 +93,28 @@ def run():
         failure = not success
     else:
         for input_file in input_files:
+            name = os.path.splitext(os.path.basename(input_file))[0]
```

Comment:
Similar to my encapsulation-related comment above, it'd be nice if this logic could be moved somewhere else - though I'm aware that's perhaps a little trickier in this case.